client-firewall-rules.md

```diff
@@ -105,6 +105,29 @@ To contact Microsoft Support: [azure.microsoft.com/support/create-ticket/](https
 
 ## Rollout Status
 
+### June 25, 2020
+
+MCR Data Endpoint changes complete!
+
+Following [Azure Safe Deployment Practices](https://azure.microsoft.com/en-us/blog/advancing-safe-deployment-practices), all MCR regional endpoints have now been updated.
+
+> **Note:** mcr uses regional CDN to provide global coverage. You will not see every Azure Data Center listed.
+> Over time, MCR regional endpoints _will change_ assuring the best performance for our customers. It is strongly recommended to use `*.data.mcr.microsoft.com` for firewall rules to avoid disruption as we alter regions for performance & readability.
+
+| Date                  | Regions         | Data Endpoints                       | Status   |
+| -                     | -               | -                                    | -        |
+| 6/12/2020 - Friday    | EUAP Central US | N/A                                  | Complete |
+| 6/13/2020 - Saturday  | EUAP East US2   | N/A                                  | Complete |
+| 6/15/2020 - Monday    | West Central US | westcentralus.data.mcr.microsoft.com | Complete |
+| 6/18/2020 - Thursday  | West US         | westus.data.mcr.microsoft.com        | Complete |
+| 6/22/2020 - Monday    | East US         | eastus.data.mcr.microsoft.com        | Complete |
+|                       | North Europe    | northeurope.data.mcr.microsoft.com   | Complete |
+|                       | Southeast Asia  | southeastasia.data.mcr.microsoft.com | Complete |
+| 6/24/2020 - Wednesday | West US2        | westcentralus.data.mcr.microsoft.com | Complete |
+|                       | Central US      | centralus.data.mcr.microsoft.com     | Complete |
+|                       | East Asia       | eastasia.data.mcr.microsoft.com      | Complete |
+|                       | West Europe     | westeurope.data.mcr.microsoft.com    | Complete |
+
 ### June 19, 2020
```

Comment:
Should this part be removed now?

Reply:
Two tables of the same kind might be confusing.

Reply:
ahh, I was updating the single entry, but it was weird that it showed a rollout status, with incremental dates, and I kept changing a single date. Seemed inconsistent with a rolling status.
In a few weeks, when we're confident we won't need to rollback, we can cleanup all these references, including the `*.cdn.mscr.io` domains, and leave it all to git history...